common/crypto_utils_openssl/avalon_crypto_utils/crypto/crypto.i

```diff
@@ -0,0 +1,154 @@
+/* Copyright 2018 Intel Corporation
```

Comment:
Is it newly created file/rename/moved file?

Reply:
It is a old file from crypto_utils. It is copied, so as to keep both cpp openssl based and pyton libraries based crypto functions.